enum/reklamationsgrund/reklamationsgrund.go

```diff
@@ -0,0 +1,13 @@
+package reklamationsgrund
+
+// Reklamationsgrund gibt den Grund einer bo.Energiemenge n- bo.Reklamation
+//go:generate stringer --type Reklamationsgrund
+//go:generate jsonenums --type Reklamationsgrund
+type Reklamationsgrund int
```

Comment:
auch hier: wenn sich das enum (auch in zukunft) nur auf Energiemengen bezieht, dann sollte das imho im namen klar werden. falls das enum generellere anwendung finden soll, dann sollten seine member (zu hoch/zu niedrif) irgendwie im namen klar machen, dass sie sich auf die energiemenge beziehen.